### DISCUSSION TITLE
Node ref warning message

### DISCUSSION THREAD

**Discussion by drazen-todorovic** [leptos-rs/leptos#2122] [Q&A]
Dear,

I am getting the following message in the web browser console:

```
You are setting a NodeRef that has already been filled. It’s possible this is intentional, but it’s also possible that you’re accidentally using the same NodeRef for multiple _ref attributes.
```

I am using three input elements with three create_node_ref variables. Is it possible that I am doing something wrong?

**Comment by gbj** [comment]
It's possible! Have any code you can share?
